fbw-a380x/src/systems/instruments/src/AtcMailbox/AtcMailbox.tsx

```diff
@@ -0,0 +1,51 @@
+//  Copyright (c) 2024 FlyByWire Simulations
+//  SPDX-License-Identifier: GPL-3.0
+
+import { DisplayComponent, EventBus, FSComponent, Subject, VNode } from '@microsoft/msfs-sdk';
+import { Button } from 'instruments/src/MFD/pages/common/Button';
+import { MouseCursor } from 'instruments/src/MFD/pages/common/MouseCursor';
+import { CdsDisplayUnit, DisplayUnitID } from '../MsfsAvionicsCommon/CdsDisplayUnit';
+
+import './style.scss';
+
+export interface AtcMailboxProps {
+  readonly bus: EventBus;
+}
+
+export class AtcMailbox extends DisplayComponent<AtcMailboxProps> {
+  private readonly topRef = FSComponent.createRef<HTMLDivElement>();
+
+  private readonly mouseCursorRef = FSComponent.createRef<MouseCursor>();
+
+  public onAfterRender(node: VNode): void {
+    super.onAfterRender(node);
+
+    this.topRef.instance.addEventListener('mousemove', (ev) => {
```

Comment:
This needs to be cleaned up in `destroy`. You'll need to move the handler to an arrow function in a class property to do that.

Reply:
Just pushed a commit, like so?